common/services/allocation.test.js

```diff
@@ -78,6 +82,26 @@ const mockAllocations = [
 
 describe('Allocation service', function () {
   describe('cancel', function () {
+    context.skip('without data args', function () {
```

Comment:
The two `without data args` tests now fail in a strange way, so have been skipped.

Are these providing value, and if so, what should the response be for a invocation missing parameters?

```
 1) Allocation service
       cancel
         without data args
           "before each" hook for "should call update method with data":
     Error: Not Found
      at error (common/lib/api-client/middleware/errors.js:13:19)
      at processTicksAndRejections (internal/process/task_queues.js:93:5)

  2) Move Service
       #cancel()
         with move ID
           without data args
             should call update method with data:
     AssertionError: expected post to have been called exactly once with exact arguments {
  cancellation_reason: undefined,
  cancellation_reason_comment: undefined,
  id: "b695d0f0-af8e-4b97-891e-92020d6820b9",
  timestamp: "#timestamp"
}
{
  cancellation_reason: undefined,
  cancellation_reason_comment: undefined,
  id: "b695d0f0-af8e-4b97-891e-92020d6820b9",
  timestamp: "#timestamp"
}
      at Context.<anonymous> (common/services/move.test.js:1056:40)
```

Reply:
Maybe they can be removed?

Reply:
Done! :)